static/src/js/components/GranuleDetails/GranuleDetailsMetadata.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './GranuleDetailsMetadata.scss'
+
+export const GranuleDetailsMetadata = ({ metadataUrls }) => {
+  const metdataUrlKeys = [
+    'native',
+    'umm_json',
+    'atom',
+    'echo10',
+    'iso19115'
+  ]
+  return (
+    <div className="granule-details-metadata">
+      <div className="granule-details-metadata__content">
+        <h4 className="granule-details-metadata__heading">Download Metadata:</h4>
+        <ul className="granule-details-metadata__list">
+          {
+            metdataUrlKeys.length && metdataUrlKeys.map((key) => {
```

Comment:
Why not do `Object.keys(metadataUrls)`?

Reply:
I did it this way so we could order them properly. Using object keys outputs them in a different order than is used currently on prod.

Reply:
Ah, ok. We define that list in a couple of places, or at least pretty close to the same list in `util/granules.js`. Any way to consolidate that code?